WordPress/src/main/java/org/wordpress/android/util/QuickStartUtils.kt

```diff
@@ -176,15 +185,43 @@ class QuickStartUtils {
         ) {
             val siteId = site.id.toLong()
 
-            if (quickStartStore.getQuickStartCompleted(siteId) || isEveryQuickStartTaskDone(quickStartStore)) {
+            if (quickStartStore.getQuickStartCompleted(siteId) || isEveryQuickStartTaskDone(quickStartStore) ||
+                    quickStartStore.hasDoneTask(siteId, task)) {
                 return
             }
 
             quickStartStore.setDoneTask(siteId, task, true)
+            AnalyticsTracker.track(getTaskCompletedTracker(task))
 
             if (isEveryQuickStartTaskDone(quickStartStore)) {
+                AnalyticsTracker.track(AnalyticsTracker.Stat.QUICK_START_COMPLETE_ALL_TASKS)
                 dispatcher.dispatch(SiteActionBuilder.newCompleteQuickStartAction(site))
             }
         }
+
+        @JvmStatic
+        fun getQuickStartListTappedTracker(task: QuickStartTask): Stat {
+            return when (task) {
+                CREATE_SITE -> AnalyticsTracker.Stat.QUICK_START_LIST_CREATE_SITE_TAPPED
+                VIEW_SITE -> AnalyticsTracker.Stat.QUICK_START_LIST_VIEW_SITE_TAPPED
+                CHOOSE_THEME -> AnalyticsTracker.Stat.QUICK_START_LIST_BROWSE_THEMES_TAPPED
+                CUSTOMIZE_SITE -> AnalyticsTracker.Stat.QUICK_START_LIST_CUSTOMIZE_SITE_TAPPED
+                SHARE_SITE -> AnalyticsTracker.Stat.QUICK_START_LIST_ADD_SOCIAL_TAPPED
+                PUBLISH_POST -> AnalyticsTracker.Stat.QUICK_START_LIST_PUBLISH_POST_TAPPED
+                FOLLOW_SITE -> AnalyticsTracker.Stat.QUICK_START_LIST_FOLLOW_SITE_TAPPED
```

Comment:
We can remove the unnecessary `AnalyticsTracker` qualification from all the `AnalyticsTracker.Stat.STAT_NAME` for simplicity.